### ISSUE TITLE
Linux documentation does not accurately reflect build requirements.

### ISSUE THREAD

**Issue by ellisgeek** [hadouken/hadouken#239]
The documentation for Ubuntu / Linux does not accurately reflect the requirements to build both boost and libtorrent.

Boost (at least per the install script) requires `gcc-4.8` and `g++-4.8` to compile with the included script. The script should either be generalized to support GCC and G++ versions greater than 4.8 (Ubuntu currently defaults to 5.1.x as of 15.10) or the specific version of GCC and G++ needed should be specified as requirements or installed via the script in `./linux`.

Libtorrent requires `automake` and `libtool` in order to build and these are not listed as dependencies in the Hadouken documentation. They should either be listed or installed as part of the libtorrent script.

**Comment by vktr** [comment]
I think the build documentation for Linux was written when we included libtorrent sources in Hadouken instead of linking against a pre-built version.

**Comment by nwgat** [comment]
i agree with @ellisgeek its almost impossible to compile, btw the deb packages are missing?
